test/functional/wallet_basic.py

```diff
@@ -587,7 +587,7 @@ def run_test(self):
 
         # Get all non-zero utxos together
         chain_addrs = [self.nodes[0].getnewaddress(), self.nodes[0].getnewaddress()]
-        singletxid = self.nodes[0].sendtoaddress(chain_addrs[0], self.nodes[0].getbalance(), "", "", True)
+        singletxid = self.nodes[0].sendall(recipients=[chain_addrs[0]])['txid']
```

Comment:
This might be out of scope, although it still uses the `sendall` to simplify the logic and makes L591-596 redundant:

```suggestion
        # Split all non-zero utxos into two chains
        self.nodes[0].sendall(recipients=chain_addrs)['txid']
```
_Note: `recipients` receive equal share of the unspecified amount_

Reply:
Nice catch! Added this on a new commit.